app/models/note_reaction.rb

```diff
@@ -0,0 +1,12 @@
+class NoteReaction < ApplicationRecord
+  enum reaction_name: { empowered: 0, strong: 1, knowledable: 2, magical: 3, peaceful: 4 }
```

Comment:
I think it'd be good if the persisted enum values were the strings themselves, rather than integers – this way any database backups or inspecting db data directly will be more readable.

Reply:
Also: could you add a presence validation for `reaction_name`.

Reply:
Done :)